service-commands/scripts/provision-dev-env.sh

```diff
@@ -12,22 +12,22 @@ function setup_linux_toolchains() {
     sudo apt update
     sudo apt-get -y upgrade
     sudo apt install -y \
-        apt-transport-https \
```

Comment:
Please replace the hanging indents since it's the "Docker" friendly way to write bash scripts.